File: level1/ex52.js
function solution(d, budget) {
  let answer = 0;
  let sum = 0;

  // 오름차순 정렬
  d.sort((a, b) => a - b);

  // 예산이 넘기전까지 개수 세기
  for (let i = 0; i < d.length; i++) {
    if (sum + d[i] > budget) break;

    sum += d[i];
    answer++;
  }

  return answer;
}

File: level1/ex52.py
def solution(depts, budget):
    # 신청 금액 오름차순 정렬
    depts = sorted(depts)
    
    # 최대 지원 가능 수
    count = 0

    for dept in depts:
        # 지원 가능한 경우
        if budget - dept >= 0:
            count += 1
            budget -= dept
        # 지원 불가능한 경우    
        else:
            break
            
    return count